exporters/jaeger/build.gradle.kts

```diff
@@ -17,12 +17,13 @@ dependencies {
 
   implementation("io.grpc:grpc-protobuf")
   implementation("io.grpc:grpc-stub")
+  implementation("com.fasterxml.jackson.jr:jackson-jr-objects")
```

Comment:
any idea how big this is compared to gson/protobuf-java-util?

Reply:
The `jackson-jr-objects-2.12.4.jar` artifact is ~[98kb](https://repo1.maven.org/maven2/com/fasterxml/jackson/jr/jackson-jr-objects/2.12.4/). `jackson-jr-objects-2.12.4.jar` depends on `jackson-core-2.12.4.jar` which is ~[365kb](https://repo1.maven.org/maven2/com/fasterxml/jackson/core/jackson-core/2.12.4/). 

The `protobuf-java-util-3.17.3.jar` artifact is ~[75kb](https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java-util/3.17.3/). `protobuf-java-util-3.17.3.jar` depends on `protobuf-java-3.17.3.jar` (amongst [other things](https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java-util/3.17.3/protobuf-java-util-3.17.3.pom)) which is is ~[1.7mb](https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/3.17.3/)